src/bank_machine/content.clj

```diff
@@ -21,51 +22,30 @@
   this structure.
   @param vector[string]
   @return vector[vector[string]]"
-
-  (def count-of-lines-of-content (count contents))
-  (loop [
-          i 0
-          account-line-collection []
-        ]
-    (if (< i count-of-lines-of-content)
-      (do
-        (def row1 (get contents i))
-        (def row2 (get contents (+ i 1)))
-        (def row3 (get contents (+ i 2)))
-        (def row4 (get contents (+ i 3)))
-        (recur (+ i 4)(conj account-line-collection [row1 row2 row3 row4]))
-      )
-      account-line-collection
-    )
-  )
-)
+  (loop [acc []
+         lines contents]
+    (if (seq lines)
```

Comment:
Another thing you'll see in Clojure code a lot is using `seq` to check that there is still data left in a collection. It's equivalent, in spirit, to `(not-empty? lines)` in this case, and forces the positive case into the first clause.